infracost-usage-example.yml

```
@@ -476,6 +476,13 @@ resource_usage:
     memory_mb: 128 # Average amount of memory consumed by function in MB. Only applicable for Consumption plan.
     instances: 2 # Number of instances. Only applicable for Premium plan.
 
+  azurerm_cosmosdb_cassandra_keyspace.my_cassandra:
+    storage_gb: 1000 # Total size of storage in GB.
+    monthly_serverless_request_units: 10000000 # Monthly number of serverless request units.
+    restored_data_gb: 3000 # Size of point-in-time restore data in GB.
```

I think this should be monthly since it's a non-continuous resources, meaning it's the total amount of data that was restored in each month (whereas storage_gb is used continuously in time so it doesn't make sense to define it as as monthly).
```suggestion
    monthly_restored_data_gb: 3000 # Monthly total amount of point-in-time restore data in GB.
```